cumulus_library/studies/core/core_templates/encounter.sql.jinja

```diff
@@ -51,6 +104,12 @@ WITH temp_encounter_nullable AS (
             ) 
         }}
     FROM encounter AS e
+    LEFT JOIN temp_encounter_completion AS tec USING(id)
+    WHERE (
+        -- NULL completion just means it's a row that isn't completion-tracked
+        -- (likely a legacy row), so allow it in.
+        tec.is_complete IS NULL OR tec.is_complete
+    )
```

Comment:
i think there's a use case for 'how many of my encounters are complete' that this is filtering out - not sure the best way to handle, just observing.

Reply:
We talked about this off-PR - it's a good point that ignored encounters have no one to speak for them with this current state.

So I should probably add a `core__incomplete_encounters` table or some such that can just hold id values - so at least if there is a mistake and someone forgot to load Conditions for a group, you'll know you've got a mistake.

When you're done with your ingestion process, and there are ids in that table, something went wrong.

Reply:
This is done now :+1: